adapt-and-extend/theming/resourceregistry.rst

```diff
@@ -18,47 +18,75 @@ Introduction to Plone 5 resources
 
 Plone 5 introduces new concepts for working with JavaScript and CSS in Plone.
 
-We finally have a solution to define JavaScript modules and their dependencies - in contrast to the previous approach, where all dependencies had to be loaded in order.
-We achieve that with the tool RequireJS, which follows the Asynchronous Module Definition (AMD) approach.
-We chose AMD over other module loading implementations (like CommonJS) because AMD can be used in non-compiled form in the browser and also compiled Through-The-Web.
-This way we can also support a "development mode" like in Plone versions prior to 5, where changes in the JavaScript sources are instantly reflected (actually they are made available a few seconds after initial browser load).
+JavaScript and AMD
+------------------
+
```

Comment:
@ebrehault I'd be happy to do so.  Can you find an example that does so so that I can point to a clean usage pattern?  Right now all the examples I've seen that do not use the AMD pattern are simply defaulting to using `jsregistry.xml` to get into the `plone-legacy` bundle